bin/acm

```diff
@@ -5,23 +5,93 @@ require 'thor'
 
 class AkamaiCloudletManagerCli < Thor
 
-  desc "clone_version", "Clones the current version"
+  desc "get_policy_version", "Get policy versions"
   long_desc <<-LONGDESC
-  Clones the current activated version. All new updates are performed in cloned version
+  Get the policy versions.
+    With --policy_id, policy on which version to be cloned
     LONGDESC
-  def clone_version
-    puts "clones the version"
+  option :policy_id, required: true
+  def get_policy_version
+    puts "Policy version:"
+    policy_version = AkamaiCloudletManager::PolicyVersion.new({policy_id: options[:policy_id] })
+    puts policy_version.versions
   end
 
-  desc "update_version", "Updates the version"
+  desc "clone_policy_version", "Clones the current policy version"
   long_desc <<-LONGDESC
-  Updates the version
-    With --version, version which needs to be updated
+  Clones the current activated version of a policy. All new updates are performed in cloned version
+    With --policy_id, policy on which version to be cloned
+    LONGDESC
+  option :policy_id, required: true
+  def clone_policy_version
+    puts "clones the policy version"
+    policy_version = AkamaiCloudletManager::PolicyVersion.new({policy_id: options[:policy_id] })
+    begin
+      latest_version = JSON.parse(policy_version.versions).first["version"].to_s
```

Comment:
Should put a check incase no policy exist
```ruby
if !policy_version.versions.empty?
  latest_version = JSON.parse(policy_version.versions).first["version"].to_s
else
  puts "No versions available"
end
```

Reply:
This sounds right, it's case where policy is just created without version, not sure whether it is possible though, but I will add this case 👍